packages/components/src/QualityBar/QualityBar.component.js

```diff
@@ -1,16 +1,57 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useTranslation } from 'react-i18next';
+
 import RatioBar from '../RatioBar';
 import {
 	QualityEmptyLine,
 	QualityInvalidLine,
 	QualityNotApplicableLine,
 	QualityPlaceholderLine,
-	QualityType,
 	QualityValidLine,
+	QualityType,
 } from './QualityRatioBar.component';
-import I18N_DOMAIN_COMPONENTS from '../constants';
+import { StackHorizontal } from '@talend/design-system';
+import { omit } from 'lodash';
+
+function QualityBarRatioBars({
```

Comment:
Created this one that assembles the various segments (in order to compute %s only once)